flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/test/java/org/apache/flink/cdc/connectors/base/source/assigner/state/PendingSplitsStateSerializerTest.java

```diff
@@ -55,14 +55,14 @@ public void testPendingSplitsStateSerializerAndDeserialize() throws IOException
                 new PendingSplitsStateSerializer(constructSourceSplitSerializer());
         PendingSplitsState streamSplitsStateAfter =
                 pendingSplitsStateSerializer.deserializePendingSplitsState(
-                        6, pendingSplitsStateSerializer.serialize(streamPendingSplitsStateBefore));
+                        7, pendingSplitsStateSerializer.serialize(streamPendingSplitsStateBefore));
         Assert.assertEquals(streamPendingSplitsStateBefore, streamSplitsStateAfter);
 
         SnapshotPendingSplitsState snapshotPendingSplitsStateBefore =
                 constructSnapshotPendingSplitsState(AssignerStatus.NEWLY_ADDED_ASSIGNING);
         PendingSplitsState snapshotPendingSplitsStateAfter =
                 pendingSplitsStateSerializer.deserializePendingSplitsState(
-                        6,
```

Comment:
Please add the test for version 6 in `testPendingSplitsStateSerializerCompatibility`.

Reply:
Adjustment completed